### PR TITLE
Update Lambda Chat docs

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**
## Description

This PR updates the Lambda Chat docs in this project to sync with the [live Lambda Chat docs](https://docs.lambdalabs.com/on-demand-cloud/using-the-lambda-chat-completions-api).

This PR also:

- Moves the Lambda Chat docs to a more fitting location, namely, under Public Cloud.
- Updates related redirects, and related links throughout the project.
- Updates _unrelated_ redirects, and _unrelated_ links throughout the project.
- Removes trailing whitespace where found.

## Related issues

None

## Checklist

- [x] Documentation follows the [contributing guidelines](../CONTRIBUTING.md).
- [x] Documentation linked to in nav and index pages.
- [ ] ~~Files used in this PR (e.g., images and PDFs) are committed.~~
- [x] Link to existing documentation and the reverse.
- [x] Changes have been tested locally.
- [x] Reviewers have been assigned (at least 1 tech writer and 1 eng).

## Additional notes

None

### CODE REVIEW

File: docs/public-cloud/1-click-clusters/index.md [hunk 1]
```diff
@@ -11,39 +11,39 @@ tags:
...
     1. *SSH Key:* Select the SSH key that you want to use to access your cluster, then click **Next**.
     1. *Cluster name:* Enter a name for your cluster, then click **Continue to Payment**. You’re presented with a billing summary and the billing information you have saved to your Lambda On-Demand Cloud account.
-1. If needed, update your billing information and change the email address where you’d like the invoice for your 1CC reservation to be sent.  
-1. Only the invoice will be sent to this email address. All notifications and other information regarding your 1CC will be sent to the email address associated with your account.  
-1. Review the listed policies and terms of service. If you agree to the policies and terms of service, click the checkbox under **Terms and Conditions**.  
+1. If needed, update your billing information and change the email address where you’d like the invoice for your 1CC reservation to be sent.
+1. Only the invoice will be sent to this email address. All notifications and other information regarding your 1CC will be sent to the email address associated with your account.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/index.md:38]: Should be a note, not a list item

File: docs/public-cloud/1-click-clusters/index.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Your 1CC will automatically launch on the day your reservation begins. You’ll
 
 ## Accessing your 1-Click Cluster
 
-Your 1CC’s management nodes have public IP addresses and can be accessed directly via SSH. You can access your compute nodes via SSH using a management node as a jump box. You can also access both your management nodes and your compute nodes using [Jupyter Notebook](../on-demand-cloud/getting-started#how-do-i-open-jupyter-notebook-on-my-instance).  
+Your 1CC’s management nodes have public IP addresses and can be accessed directly via SSH. You can access your compute nodes via SSH using a management node as a jump box. You can also access both your management nodes and your compute nodes using [Jupyter Notebook](../on-demand/getting-started.md#how-do-i-open-jupyter-notebook-on-my-instance).
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/index.md:55]: Nit: change smart-quote apostrophe to regular apostrophe

```suggestion
Your 1CC's management nodes have public IP addresses and can be accessed directly via SSH. You can access your compute nodes via SSH using a management node as a jump box. You can also access both your management nodes and your compute nodes using [Jupyter Notebook](../on-demand/getting-started.md#how-do-i-open-jupyter-notebook-on-my-instance).
```

File: docs/public-cloud/lambda-chat-api.md [hunk 3]
```diff
@@ -8,47 +8,60 @@ tags:
...
-
-- `hermes-3-llama-3.1-405b-fp8` (18K context length)
-- `hermes-3-llama-3.1-405b-fp8-128k` (128K context length)
+- [Creating chat completions](#creating-chat-completions) (`/chat/completions`)
+- [Creating completions](#creating-completions) (`/completions`)
+- [Listing models](#listing-models) (`/models`)
+
+Currently, the following models are available:
+
+- `deepseek-coder-v2-lite-instruct`
```
Comment (ryanlambda) [inline docs/public-cloud/lambda-chat-api.md:38]: Non-blocking suggestion: Turn this into a table eventually

**Review by ryanlambda — CHANGES_REQUESTED** [review]
Generally LGTM. Just need to fix the following:

- The erroneous list item in `1-click-clusters/index.md`
- The page list on the Education landing page, which has become noticeably out of sync with the left nav

File: docs/public-cloud/1-click-clusters/index.md (hunk 2 above)
Comment (cbrownstein-lambda) [reply docs/public-cloud/1-click-clusters/index.md]: Just noting that smart quotes appear in many places:

```
$ git grep --name-only ’ \*.md
docs/education/large-language-models/serving-llama-3-1-405b.md
docs/education/linux-usage/basic-linux-commands-and-system-administration.md
docs/education/linux-usage/troubleshooting-and-debugging.md
docs/education/linux-usage/using-the-lambda-bug-report-to-troubleshoot-your-system.md
docs/education/linux-usage/using-the-nvidia-bug-report.log-file-to-troubleshoot-your-system.md
docs/education/programming/virtual-environments-containers.md
docs/education/tutorial-getting-started-with-training-a-machine-learning-model.md
docs/hardware/workstations/getting-started.md
docs/public-cloud/1-click-clusters/index.md
docs/public-cloud/1-click-clusters/support.md
docs/public-cloud/cloud-api.md
docs/public-cloud/filesystems.md
docs/public-cloud/firewalls.md
docs/public-cloud/lambda-chat-api.md
docs/public-cloud/on-demand/dashboard.md
docs/public-cloud/on-demand/demos.md
docs/public-cloud/on-demand/getting-started.md
docs/public-cloud/teams.md
```

**Comment by cbrownstein-lambda** [comment]
> Generally LGTM. Just need to fix the following:
> - The erroneous list item in 1-click-clusters/index.md

Fixed in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/47/commits/213fe709f40640b3d9f31a9896605c6ecd08c39f.

> - The page list on the Education landing page, which has become noticeably out of sync with the left nav

Fixed in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/pull/47/commits/70a9b8c227d42aae257025db81cf36fc50bc1d50.

**Review by ryanlambda — APPROVED** [review]

File: docs/public-cloud/1-click-clusters/index.md (hunk 1 above)
Comment (cbrownstein-lambda) [reply docs/public-cloud/1-click-clusters/index.md]: Fixed in https://github.com/cbrownstein-lambda/lambda-docs-mkdocs/commit/213fe709f40640b3d9f31a9896605c6ecd08c39f.

**Pull request merged**